reversed.go

```diff
@@ -0,0 +1,19 @@
+package do
+
+import "reflect"
+
+/*
+Reversed returns the values of a given interface on a reversed basis.
+*/
+func Reversed(iterable interface{}) interface{} {
+	iterValue := getIterable(iterable)
+	returnValue := reflect.MakeSlice(iterValue.Type(), 0, 0)
```

Comment:
You already know the size of the slice so no need to set it to 0 and append (very expensive)